test/functional/rpc_blockchain.py

```diff
@@ -352,6 +355,42 @@ def assert_waitforheight(height, timeout=2):
         assert_waitforheight(current_height)
         assert_waitforheight(current_height + 1)
 
+    def _test_getblock(self):
+        if not self.is_wallet_compiled():
```

Comment:
Did you look at using miniwallet.py to manually create the transactions? That would avoid a dependency on a compiled wallet to test this functionality.

Reply:
Thanks, I didn't know about `MiniWallet` before. That is very useful!